src/77480.c

```diff
@@ -603,7 +605,115 @@ void func_800E06C0(s32 arg0) {
 
 INCLUDE_ASM(s32, "77480", func_800E06D8);
 
-INCLUDE_ASM(s32, "77480", check_for_interactables);
+void check_for_interactables(void) {
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    Npc* npc = gPlayerStatus.unk_C8;
+    s32 phi_s2;
+
+    if (playerStatus->animFlags & PLAYER_STATUS_ANIM_FLAGS_100 || D_8010C940 || D_8010C920) {
```

Comment:
I know this goes against what we normally do, but I'd prefer to wrap the flag check part of this in parens to make it clearer next to those other bool checks